test_metannotate_end_to_end.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+
+"""
+Created by: Metannotate Team (2017)
+
+Description: A shell script that runs a sample test of Metannotate and compares the generated
+             output files to verify proper end-to-end functionality of the program.
+
+Requirements: - The following packages:
```

Comment:
@KyleBehiels Replace all the requirements with a simple "A full CLI installation of Metannotate."